src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-user-form/rgw-user-form.component.ts

```diff
@@ -705,4 +705,14 @@ export class RgwUserFormComponent implements OnInit {
     result = _.uniq(result);
     return result;
   }
+
+  onMaxBucketsModeChange(mode: string) {
+    if (mode === '1') {
+      // If 'Custom' mode is selected, then ensure that the form field
+      // 'Max. buckets' contains a valid value. Set it to default if
+      // necessary.
+      const maxBuckets = this.userForm.getValue('max_buckets');
+      this.userForm.patchValue({ max_buckets: _.isEmpty(maxBuckets) ? 1000 : maxBuckets });
```

Comment:
Don't think this is working correctly.
Any value set in `maxBuckets` is being overwritten with `1000`, since isEmpty is always returning `true`
e.g. `_.isEmpty(500) === true`

Reply:
Fixed